themes/gruvbox-dark-hard.json

```diff
@@ -479,6 +479,40 @@
         "foreground": "#fabd2f"
       }
     },
+    // C# ----------------------------------------
+    {
+      "name": "C# functions & namespace",
+      "scope": [
+        "meta.function.cs",
+        "entity.name.function.cs",
+        "entity.name.type.namespace.cs"
+      ],
+      "settings": {
+        "foreground": "#b8bb26"
+      }
+    },
+    {
+      "name": "C# Variables",
+      "scope": [
+        "keyword.other.using.cs",
+        "entity.name.variable.field.cs",
+        "entity.name.variable.local.cs",
+        "variable.other.readwrite.cs"
+      ],
+      "settings": {
+        "foreground": "#8ec07c"
+      }
+    },
+    {
+      "name": "C# This",
+      "scope": [
+        "keyword.other.this.cs",
+        "keyword.other.base.cs"
+      ],
+      "settings": {
+        "foreground": "#b16286"
```

Comment:
Could we try this with the light purple `#d3869b` and see how this goes?
If we try & decide to go with the light purple for dark variants, perhaps we'll need to look at dark purple `#8f3f71` for the light variants.